vulture/utils.py

```diff
@@ -11,10 +11,11 @@ class VultureInputException(Exception):
     pass
 
 
-def condition_is_unsatisfiable(condition):
+def evaluate_condition(condition):
```

Comment:
It's probably best to add two new functions that call this function: `condition_is_always_true` and `condition_is_always_false`.